src/components/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+
+import { useContext } from 'react';
+import { TodosContext } from '../../context/TodosContext';
+
+import { Footer } from '../Footer';
+import { Header } from '../Header';
+import { Main } from '../Main';
+import { filteredTodos } from '../../services/filterTodos';
+
+export const TodoApp = () => {
+  const {
+    todos,
+    errorMessage,
+    setErrorMessage,
+    filterTodos,
+  } = useContext(TodosContext);
+
+  const handleCloseError = () => {
+    setErrorMessage('');
+  };
+
+  const filteredItems = filteredTodos(todos, filterTodos);
+
+  return (
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <Header />
+
+        {todos && (
+          <Main items={filteredItems} />
+        )}
```

Comment:
todos will be always a `true` even if you have an empty array

thats why check by array length